src/components/left-sidebar/LeftSideBar.jsx

```diff
@@ -0,0 +1,99 @@
+import "./leftsidebar.css";
+import { Link, useLocation } from "react-router-dom";
+import {
+  MdOutlineHome,
+  MdHome,
+  MdOutlineExplore,
+  MdExplore,
+  MdBookmarkBorder,
+  MdBookmark,
+} from "react-icons/md/";
+import { FaRegUser, FaUser } from "react-icons/fa/";
+import { useSelector } from "react-redux";
+
+export function LeftSideBar() {
+  const location = useLocation();
+  const user = useSelector((state) => state.auth.user);
+  return (
+    <aside className="sm-left-sidebar">
+      <ul className="left-sidebar-list-container">
```

Comment:
Suggestion: you can map this by storing all the data in an array of objects, since the structure is repeating